test/contracts/SampleCrowdsale.test.js

```diff
@@ -0,0 +1,59 @@
+const Jurisdiction = artifacts.require('Jurisdiction');
+const SampleToken = artifacts.require('SampleToken');
+const SampleCrowdsale = artifacts.require('SampleCrowdsale');
+
+const BigNumber = web3.BigNumber;
+
+var should = require('chai')
+        .use(require('chai-as-promised'))
+        .use(require('chai-bignumber')(BigNumber))
+        .should();
+
+contract('SampleCrowdsale', function ([owner, wallet, investor]) {
+
+    const RATE = new BigNumber(1);
+
+    beforeEach(async function () {
+        this.jurisdiction = await Jurisdiction.new();
+        this.jurisdiction.addValidator(owner);
+
+        this.token = await SampleToken.new(this.jurisdiction.address);
+        this.crowdsale = await SampleCrowdsale.new(
+            RATE, wallet, this.token.address);
+        await this.token.transferOwnership(this.crowdsale.address);
+    });
+
+    it('should create crowdsale with correct parameters', async function () {
+        this.crowdsale.should.exist;
+        this.token.should.exist;
```

Comment:
I think these two lines are redundant... We can assume the contract constructors to return an instance.

Reply:
removed.